railties/test/application/configuration_test.rb

```diff
@@ -4203,6 +4203,44 @@ def new(app); self; end
       assert_match(/The `legacy_connection_handling` setter was deprecated in 7.0 and removed in 7.1, but is still defined in your configuration. Please remove this call as it no longer has any effect./, error.message)
     end
 
+    test "encrypted attributes are added to record's filter_attributes by default" do
```

Comment:
I'm on the fence about whether we should pose this test as "by default" or as "when add_to_filter_parameters is true".  Right now, `add_to_filter_parameters` is indeed true by default, but if that were to change, we would still want to test the case where `add_to_filter_parameters` is true.  On the other hand, the default behavior around `add_to_filter_parameters` doesn't seem to be tested anywhere else, so perhaps there is value in keeping this test as is.

Also, what do you think about moving this tests right below [`"ActiveRecord::Base.filter_attributes should equal to filter_parameters"`](https://github.com/rails/rails/blob/46608c9398a71b68e37ded3d7875bdc9c214f876/railties/test/application/configuration_test.rb#L3393-L3400)?

Reply:
> On the other hand, the default behavior around `add_to_filter_parameters` doesn't seem to be tested anywhere else, so perhaps there is value in keeping this test as is.

Yeah, seeing some similar behavior with other tests in this file so I agree there's some value in ensuring the default value doesn't change.